src/styles/utils/keyframes.scss

```
@@ -0,0 +1,9 @@
+@keyframes rotate {
+  from {
+    transform: translateY(-50%) rotate(0deg);
+  }
+
+  to {
+    transform: translateY(-50%) rotate(360deg);
+  }
+}
```

since `translateY(-50%)` isn't changed here you can move this rule directly to hands
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}
```

---

This suggestion breaks all)

---

Not resolved. If u have any questions - feel free to ask in fe_chat